templates/resources/network_device.md.tmpl

```diff
@@ -51,6 +54,7 @@ The following arguments are supported:
 * `package_code` - (Required) Device software package code.
 * `version` - (Required) Device software software version.
 * `core_count` - (Required) Number of CPU cores used by device. (**NOTE: Use this field to resize your device. When resizing your HA devices, primary device will be upgraded first. If the upgrade failed, device will be automatically rolled back to the previous state with original core number.**)
+* `tier` - (Optional) Select bandwidth tier for your own license, i.e., `0` or `1` or `2` or `3`. Tiers applicable only for C8000V Autonomous or C8000V SDWAN (controller) device types.
```

Comment:
Same here, default value will be applied for C8000V device if tier is not provided. We can put that information here.

Reply:
done